[DNM] PR test 2

bnbarham opened the pull request.

AnthonyLatsis commented:
@swift-ci please clean test Windows

AnthonyLatsis commented:
@swift-ci please clean test Windows

adrian-prantl commented:
@swift-ci test macOS

adrian-prantl commented:
@swift-ci test linux

AnthonyLatsis commented:
https://github.com/swiftlang/llvm-project/pull/11438
@swift-ci test Linux

AnthonyLatsis committed:
test amazon linux 2 preset

AnthonyLatsis commented:
https://github.com/swiftlang/llvm-project/pull/11155
@swift-ci please build toolchain amazon linux 2

AnthonyLatsis commented:
https://github.com/swiftlang/llvm-project/pull/11155
@swift-ci please build toolchain amazon linux 2

AnthonyLatsis commented:
https://github.com/swiftlang/llvm-project/pull/11155
@swift-ci please build toolchain amazon linux 2

AnthonyLatsis committed:
build-script-impl: Add flag for skipping LLDB unit tests

This will be used for a special Amazon Linux 2 preset. The Python
requirement for running LLDB unit tests is now too high for AL2.

AnthonyLatsis commented:
https://github.com/swiftlang/llvm-project/pull/11155
@swift-ci please build toolchain amazon linux 2

AnthonyLatsis commented:
https://github.com/swiftlang/llvm-project/pull/11155
@swift-ci please build toolchain amazon linux 2

AnthonyLatsis commented:
https://github.com/swiftlang/llvm-project/pull/11155
@swift-ci please build toolchain amazon linux 2
